rs/node_rewards/canister/src/metrics.rs

```diff
@@ -2,20 +2,37 @@
 use crate::KeyRange;
 use crate::chrono_utils::{first_unix_timestamp_nanoseconds, last_unix_timestamp_nanoseconds};
 use crate::pb::v1::{SubnetIdKey, SubnetMetricsKey, SubnetMetricsValue};
+#[cfg(feature = "test")]
```

Comment:
nit (not blocking):
Instead of all these conditional `use`s, you could simply use use the full scope at the place of use, within the conditionally-compiled functions. Or have a separate test-specific module. Might end up being a bit cleaner.